test/versioned-external/checkout-external-tests.js

```diff
@@ -18,7 +18,7 @@ const repos = require('./external-repos')
 
 const TEMP_TESTS_FOLDER = 'TEMP_TESTS'
 
-const CHECKOUT_FOLDERS = ['lib', 'tests/versioned']
+const CHECKOUT_FOLDERS = ['index.js', 'nr-hooks.js', 'lib', 'tests/versioned']
```

Comment:
are we really saving much time doing a sparse clone of specific files? we may have to PR this later as repos evolve

Reply:
i guess you can specify additional files, n/m